### DISCUSSION TITLE
dbgate.org

### DISCUSSION THREAD

**Discussion by blaisedegrille** [dbgate/dbgate#769] [General]
Hi,

I would like to inform you, that your domain (https://dbgate.org/
) has expired. I just noticed while i was searching for an sql client which works perfectly on Linux. 

Have a nice day,
 Balázs 

![image](https://github.com/dbgate/dbgate/assets/47615137/d4ab2566-8bdf-4b71-b0cd-c70a7c259f26)

**Comment by janproch** [comment]
Thanks, fixed
